app/src/config/feature-flags.ts

```diff
@@ -98,8 +98,13 @@ const baseHeaders = {
 const logger = Logger.getInstance('featureFlags');
 
 async function getFeatureFlag(featureFlagKey: string): Promise<FeatureFlag> {
+    const REQUEST_TIMEOUT_MILLISECONDS = 2000;
```

Comment:
2000 felt nice, its long enough but not too long but interested in others thoughts

Reply:
This might be completely useless info but their example timeouts here are 3 seconds: https://docs.launchdarkly.com/sdk/features/config